### ISSUE TITLE
Cannot read properties of null (reading 'match')

### ISSUE THREAD

**Issue by dkrnl** [csstree/stylelint-validator#49]
Hi!

after update `css-tree@2.1` to `css-tree@2.2`:

```
> stylelint-csstree-validator-bug@1.0.0 test
> npx stylelint test.css

TypeError: Cannot read properties of null (reading 'match')
    at ~/stylelint-csstree-validator-bug.localhost/stylelint-csstree-validator-bug/test.css:1:2
    at matchSyntax (~/stylelint-csstree-validator-bug.localhost/stylelint-csstree-validator-bug/node_modules/css-tree/cjs/lexer/Lexer.cjs:82:51)
    at Lexer.matchAtrulePrelude (~/stylelint-csstree-validator-bug.localhost/stylelint-csstree-validator-bug/node_modules/css-tree/cjs/lexer/Lexer.cjs:289:16)
    at ~/stylelint-csstree-validator-bug.localhost/stylelint-csstree-validator-bug/node_modules/stylelint-csstree-validator/cjs/index.cjs:123:31
    at ~/stylelint-csstree-validator-bug.localhost/stylelint-csstree-validator-bug/node_modules/postcss/lib/container.js:119:18
    at ~/stylelint-csstree-validator-bug.localhost/stylelint-csstree-validator-bug/node_modules/postcss/lib/container.js:55:18
    at Root.each (~/stylelint-csstree-validator-bug.localhost/stylelint-csstree-validator-bug/node_modules/postcss/lib/container.js:41:16)
    at Root.walk (~/stylelint-csstree-validator-bug.localhost/stylelint-csstree-validator-bug/node_modules/postcss/lib/container.js:52:17)
    at Root.walkAtRules (~/stylelint-csstree-validator-bug.localhost/stylelint-csstree-validator-bug/node_modules/postcss/lib/container.js:117:19)
    at ~/stylelint-csstree-validator-bug.localhost/stylelint-csstree-validator-bug/node_modules/stylelint-csstree-validator/cjs/index.cjs:92:14
    at ~/stylelint-csstree-validator-bug.localhost/stylelint-csstree-validator-bug/node_modules/stylelint/lib/lintPostcssResult.js:113:8

```
`npm i css-tree@2.1 --save-exact` -- this fix command

https://github.com/dkrnl/stylelint-csstree-validator-bug -- test bug repo

**Comment by lahmatiy** [comment]
Thank you for the issue! Especially for repro repo 👍 
Confirm, this is a bug which were introduced in css-tree `2.2.0`. The fix is already [landed](https://github.com/csstree/csstree/commit/de6c241697f535f10c91838d04eb50c4985b048d), will be available in the next release soon.

**Comment by lahmatiy** [comment]
Fixed in CSSTree [`v2.2.1`](https://github.com/csstree/csstree/releases/tag/v2.2.1). Please, update to latest version.
